sagemaker-triton/fil_ensemble/2_triton_xgb_fil_ensemble.ipynb

```diff
@@ -0,0 +1,908 @@
+{
```

Comment:
<p>More explicitly and prominently warn users that they must run the previous notebook in order to run this notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Added more prominent text and formatting